pkg/label/label.go

```diff
@@ -7,10 +7,12 @@ const (
 const (
 	Cluster       = "giantswarm.io/cluster"
 	Organization  = "giantswarm.io/organization"
+	Provider      = "giantswarm.io/provider"
 	VersionBundle = "giantswarm.io/version-bundle"
 )
 
 const (
 	MachineDeploymentSubnet = "machine-deployment.giantswarm.io/subnet"
 	OperatorVersion         = "aws-operator.giantswarm.io/version"
+	ReleaseVersion          = "release.giantswarm.io/version"
```

Comment:
Already in the docs. Nice! 🎉 

https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md